tide_site.services.yml

```diff
@@ -18,3 +18,8 @@ services:
     arguments: ['@entity_type.manager', '@entity.repository']
     calls:
       - [setContainer, ['@service_container']]
+  tide_site.event_subscriber:
```

Comment:
@vincent-gao @anthony-malkoun I still prefer to see a system setting to toggle the `system.admin_content` view replacement. Without the toggle, all consumer sites will always have their Admin Content view changed to the new view. I'm not sure if it's the case for DHHS.

Reply:
I will check with Jess tmr about what you concerned. thanks @sonnykt

Reply:
Thanks @sonnykt

Reply:
@sonnykt @vincent-gao I think that we're just going with a blanket solution here. All sites that use `tide_site` will get the new admin view.